src/meta/user/authorizer.ts

```diff
@@ -212,6 +212,65 @@ const canViewHistory = (props: {
 export const canViewGeo = (props: { cycle: Cycle; countryIso: AreaCode; user: User }): boolean =>
```

Comment:
It's possible to clean this file up, but I did not want to do it in this PR, added refactor issue with some samples I experimented with:

https://github.com/orgs/openforis/projects/8?pane=issue&itemId=82150817

Reply:
Changing this file requires adding unit tests